lib/client/doc.js

```diff
@@ -88,6 +88,9 @@ function Doc(connection, collection, id) {
   // This is a list of {[create:{...}], [del:true], [op:...], callbacks:[...]}
   this.pendingOps = [];
 
+  this._heldOpId = 1;
+  this._heldOps = {};
```

Comment:
I know that the other props on this class aren't prefixed, but I'd like to keep things "private" where possible